src/hotspot/share/code/codeCache.cpp

```diff
@@ -323,7 +324,7 @@ void CodeCache::initialize_heaps() {
 
   // If large page support is enabled, align code heaps according to large
   // page size to make sure that code cache is covered by large pages.
-  const size_t alignment = MAX2(ps, os::vm_allocation_granularity());
+  const size_t alignment = min_size;
```

Comment:
Just a suggestion: I think it would be clearer to simply remove `alignment` and replace it with `min_size` below.

Reply:
Thanks, removed. I also updated the comment above to clarify the connection between alignment and min_size.